README.md

```diff
@@ -9,93 +9,69 @@ Works well with polymorphic data structures.
 
 ## Install
 
-Install the [npm-package](https://www.npmjs.com/package/ngx-row-accordion):
-
-`yarn add ngx-sub-form`
-
-## Setup
-
-```diff
-+ import { NgxSubFormModule } from 'ngx-sub-form';
-
-@NgModule({
-  declarations: [AppComponent],
-  imports: [
-    BrowserModule,
-+   NgxSubFormModule
-  ],
-  bootstrap: [AppComponent],
-})
-export class AppModule {}
-```
+Install the [npm-package](https://www.npmjs.com/package/ngx-sub-form):
 
 ## Usage
 
 _Before we get started with how to use the library and give some examples, a complete demo is available on [this repo](https://github.com/cloudnc/ngx-sub-form), within the `src` folder.  
-Demo is built around a concept of galatic sales. You can sell either Droids (Protocol, Medical, Astromech, Assassin) or Vehicules (Spaceship, Speeder). This will also be used for the following examples_.
+Demo is built around a concept of galactic sales. You can sell either Droids (Protocol, Medical, Astromech, Assassin) or Vehicles (Spaceship, Speeder). This will also be used for the following examples_.
 
 ### First component level
 
-Within the component where the form will be handled, we have to define the top level structure of the form (as you'd normally do).
+Within the component where the form will be handled, we have to define the top level structure of the form, with _each
+polymorphic type having it's own form control_
 
 ```ts
-public sellForm: FormGroup = new FormGroup({
-  sell: new FormControl(null, { validators: [Validators.required] }),
+public listingForm: FormGroup = new FormGroup({
+  droidListing: new FormControl(null, Validators.required),
+  vehicleListing: new FormControl(null, Validators.required),
+  listingType: new FormControl(null, Validators.required)
 });
 ```
 
-Then we need to create a separated `FormControl` to select the sell's type:
-
-```ts
-public selectSellType: FormControl = new FormControl();
-```
-
 and give access to our `enum` from the component:
 
 ```ts
-public SellType = SellType;
+public ListingType = ListingType;
 ```
 
-Just as a sidenote here, here's the `SellType` enum:
+Just as a sidenote here, here's the `ListingType` enum:
 
 ```ts
-export enum SellType {
-  VEHICULE = 'Vehicule',
+export enum ListingType {
+  VEHICLE = 'Vehicle',
   DROID = 'Droid',
 }
 ```
 
-Then, within the `.html` we create a `select` tag to choose between the 2 types:
+Then, within the `.component.html` we create a `select` tag to choose between the 2 types:
 
 ```html
-<select [formControl]="selectSellType">
-  <option *ngFor="let sellType of SellType | keyvalue" [value]="sellType.value">
-    {{ sellType.value }}
+<select [formControlName]="formControlNames.listingType">
```

Comment:
Before jumping on `formControlNames` we probably need to show that the component is extending `NgxSubFormComponent`